app/src/main/java/at/roteskreuz/stopcorona/model/manager/ChangelogManager.kt

```diff
@@ -0,0 +1,93 @@
+package at.roteskreuz.stopcorona.model.manager
+
+import android.content.SharedPreferences
+import android.text.SpannableStringBuilder
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import at.roteskreuz.stopcorona.R
+import at.roteskreuz.stopcorona.constants.Constants.Prefs.CHANGELOG_MANAGER_PREFIX
+import at.roteskreuz.stopcorona.model.repositories.other.ContextInteractor
+import at.roteskreuz.stopcorona.skeleton.core.utils.intSharedPreferencesProperty
+
+/**
+ * Manages changelog information.
+ */
+interface ChangelogManager {
+
+    /**
+     * Checks if there is an unseen changelog for the given [version].
+     */
+    fun unseenChangelogForVersionAvailable(version: String): Boolean
+
+    /**
+     * Get the changelog for the given [version].
+     *
+     * @return null when there is no changelog for the given [version]
+     * or the changelog has already been displayed.
+     */
+    fun getChangelogForVersion(version: String): Changelog?
+}
+
+class ChangelogManagerImpl(
+    preferences: SharedPreferences,
+    private val contextInteractor: ContextInteractor
+) : ChangelogManager {
+
+    companion object {
+        private const val PREF_LAST_SEEN_CHANGELOG_ID = CHANGELOG_MANAGER_PREFIX + "last_seen_changelog_id"
+    }
+
+    /**
+     * Changelog content for a list of specific versions.
+     *
+     * [Changelog.id] has to be increased for every new changelog.
+     * [Changelog.versions] contains the list of versions this changelog applies to.
+     */
+    private val changelog = Changelog(
+        id = 1,
+        versions = listOf("2.0.0"),
+        title = R.string.changelog_title_v2_0_0,
+        description = with(SpannableStringBuilder()) {
```

Comment:
Spannable is part of Android framework. It would be nicer to not have this connection in the Model layer. But feel free to keep it.

Reply:
done